Dockerfile

```diff
@@ -6,8 +6,6 @@ WORKDIR /usr/src/app/
 COPY engine.json /
 COPY package.json /usr/src/app/
 
-ENV NODE_ENV production
```

Comment:
We were packaging `node_modules` up along in the Docker image. I added it to `.dockerignore` so it stopped doing so.